template/script.sh.erb

```diff
@@ -86,16 +86,33 @@ else
     BINDS=/data,${SERVER_DATA_DIR}/var/lib:/var/lib/rstudio-server,${SERVER_DATA_DIR}/var/run:/var/run/rstudio-server
 fi
 
-singularity run --no-home -B "$BINDS" \
-  "$RSTUDIO_SERVER_IMAGE" \
-  --www-port "${port}" \
-  --auth-none 0 \
-  --auth-timeout-minutes 0 \
-  --auth-stay-signed-in-days 30 \
-  --auth-pam-helper-path "${RSTUDIO_AUTH}" \
-  --auth-encrypt-password 0 \
-  $(if [[ ! -z "${RSTUDIO_WHICH_R}" ]]; then echo --rsession-which-r "${RSTUDIO_WHICH_R}"; fi) \
-  --rsession-path "${RSESSION_WRAPPER_FILE}" \
-  --server-data-dir='/tmp/run' \
-  --server-user=$USER
- 
+if [[ "$RSTUDIO_SERVER_IMAGE" = *singularity-rstudio* ]]; then
+  singularity run --no-home -B "$BINDS" \
```

Comment:
shouldn't this also be apptainer then?

Reply:
Left like this to make the two calls more distinguishable. This block will be remove anyway after "beta" is over.